### PR TITLE
added error styling

### PR DESCRIPTION
**Pull request by Clearwood**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

Readded error styling to the country selection.

### CODE REVIEW

**Comment by Clearwood** [comment]
@troggy as an addition to issue #142, here is the correct error styling

**Comment by troggy** [comment]
I suggest to make the select the same width as text fields (keeping the text alignment). Basically removing these gaps, so that red outline is visually applied to the select box itself
image

**Comment by Clearwood** [comment]
image
image

**Comment by troggy** [comment]
amazing 👏

File: packages/burner-plugin/src/ui/topupPage/components/CountryDropdown.tsx [hunk 1]
```diff
@@ -7,26 +7,27 @@ import { ErrorMessage } from "./FormField";
...
   height: 2.5em;
-  padding: 0.4em 0.6em;
-
   background-color: white;
   max-width: 334px;
   margin-top: 0.3em;
 `;
 
+const Select = styled.select`
+  width: 100\%;
```
Comment (troggy) [inline packages/burner-plugin/src/ui/topupPage/components/CountryDropdown.tsx:18]:
```suggestion
  width: 100%;
```

**Pull request merged**